.gitignore

```diff
@@ -6,6 +6,11 @@ gradle.properties
 
 # Ignore Gradle build output directory
 build
+<<<<<<< HEAD
```

Comment:
Did you mean to change this file?

Reply:
Yes, I supposed to add bin and target (the former generated by gradle, the latter by maven)

Reply:
Got it. I am wondering why the `<<<<<` and `>>>>` diff lines are in there though?